base-math/truncdddf.c

```diff
@@ -66,15 +66,25 @@ CONVERT_WRAPPER(
 	          return SIGNBIT(a) ? -INFINITY : INFINITY;
 	      }
 	  }
-	else if (exp < -BINPOWOF10_LIMIT)	/* Obvious underflow.  */
+	else if (exp <= -BINPOWOF10_DENORM_DBL)	/* Obvious underflow.  */
 	  {
 	    if (DFP_EXCEPTIONS_ENABLED)
 	      DFP_HANDLE_EXCEPTIONS (FE_UNDERFLOW|FE_INEXACT);
 
 	    switch (fegetround())
 	      {
 	        case FE_TONEAREST:
-	          return SIGNBIT(a) ? -__DBL_DENORM_MIN__ : __DBL_DENORM_MIN__;
+	          mant = llabs (a_norm);
+	          if (exp < -BINPOWOF10_DENORM_DBL ||
+	                    ((exp == -BINPOWOF10_DENORM_DBL) &&
+	                     (mant >= 24703282292062300)))
+	           {
+	              return SIGNBIT(a) ? -0.0 : 0.0;
+	           }
```

Comment:
Wrong indentation here. You should get rid of {} and indent the return with 2 spaces.

Reply:
Will be fixed.